graphx/src/test/scala/org/apache/spark/graphx/lib/ShortestPathsSuite.scala

```diff
@@ -40,7 +40,7 @@ class ShortestPathsSuite extends FunSuite with LocalSparkContext {
       val graph = Graph.fromEdgeTuples(edges, 1)
       val landmarks = Seq(1, 4).map(_.toLong)
       val results = ShortestPaths.run(graph, landmarks).vertices.collect.map {
-        case (v, spMap) => (v, spMap.mapValues(_.get))
```

Comment:
It does look like the get is redundant, since Int should be sufficient for this purpose.